addons/assistance.py

```diff
@@ -697,8 +697,8 @@ def check_console(self, message, channel, consoles):
                                 • When the console powers off from the normal OS if AutoRCM is installed, it will automatically reboot to RCM. \
 You would need to inject Hekate and select `Power off` from the menu to fully turn the console off.
                                 
-                                • You will depend on your SD Card to take advantage of CFW. If anything happens to it\
-or your card slot, your console will only be able to boot stock firmware. \
+                                • You will depend on your SD Card to take advantage of CFW. In the event that the microSD is unavailable, you can only boot into stock firmware. \
+                                `**Warning: if you updated from <4.1.0 without burning fuses, booting stock firmware __will__ update your game card slot, rendering it unusable when you restore a <4.1.0 NAND!!`
 
                                 • The console doesn't charge quickly in RCM. To gain normal charge speed, charge to 1.5%, inject Hekate+Atmosphère.\
```

Comment:
Suggested line: "The console charges very slowly in RCM. To regain normal charging speed, charge to 1.5% and then inject Hekate and boot into Atmosphère."
slight rewording of line 703, doesnt change the meaning but its easier to read overall